apis/v1/jaeger_webhook.go

```diff
@@ -96,8 +97,13 @@ func (j *Jaeger) ValidateCreate() (admission.Warnings, error) {
 func (j *Jaeger) ValidateUpdate(_ runtime.Object) (admission.Warnings, error) {
 	jaegerlog.Info("validate update", "name", j.Name)
 
-	if len(j.Spec.Agent.Options.opts) != 0 {
-		return nil, fmt.Errorf("Jaeger agent configuration is no longer supported. See https://github.com/jaegertracing/jaeger/issues/4739. Values: %+v", j.Spec.Agent.Options.opts)
+	got, err := json.Marshal(j.Spec.Agent)
```

Comment:
Would it make sense to add an E2E test to check that, when you add something to the agent spec the Jaeger instance creation fails?

Reply:
Iam not sure if a e2e is needed.